BUILD/phylums/banish.dat

```diff
@@ -0,0 +1,12 @@
+dude	loc:The Black Forest
+dude	loc:Twin Peak
+dude	loc:The Red Zeppelin;sniffed:Red Butler;item:Glark Cable>2
```

Comment:
banishes override tracks (after nerf day 2024), so sniffing the red butler won't help.